### DISCUSSION TITLE
How-To: PasswordPusher on Synology NAS via Container Manager Project

### DISCUSSION THREAD

**Discussion by davidande** [pglombardo/PasswordPusher#2742] [Show and tell]
Hello,
I have made a configuration of PasswordPusher Docker on my Synology NAS via Container Manager. [https://github.com/davidande/passwordpusher_docker_synology](url).
Passwords, Files, Urls, smtp configuration, local storage...
Hope it can help
David

**Comment by pglombardo** [comment]
Hi @davidande - Thanks for the contribution!  I'll add a pointer in the docs to your repository under a new Docker Synology NAS section.

**Reply by pglombardo** [reply]
A link to the repository has been added to the documentation portal:
https://docs.pwpush.com/docs/installation/#synology-nas

Thanks @davidande!
